mrchecker-framework-modules/mrchecker-database-module/src/test/java/com/capgemini/mrchecker/database/unit/base/environment/EnvironmentParamTest.java

```diff
@@ -0,0 +1,66 @@
+package com.capgemini.mrchecker.database.unit.base.environment;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNull.notNullValue;
+
+import org.junit.jupiter.api.Test;
+
+import com.capgemini.mrchecker.database.core.base.environment.EnvironmentParam;
+import com.capgemini.mrchecker.database.tags.UnitTest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@UnitTest
+public class EnvironmentParamTest {
+
+	private final String ENV = System.getProperty("env", "DEV");
+
+	private static final Map<String, String> userEnvMapping = new HashMap<String, String>(){
+		{
+			put("DEV", "tester");
+			put("QA", "user");
+		}
+	};
+
+	private static final Map<String, String> passwordEnvMapping = new HashMap<String, String>(){
+		{
+			put("DEV", "Vx4hylOJhNzRc5kKm4gr");
+			put("QA", "pass");
+		}
+	};
+
+	private static final Map<String, String> connectionUrlEnvMapping = new HashMap<String, String>(){
+		{
+			put("DEV", "localhost:3307/int_tests");
+			put("QA", "de-muc-devonfw-allure01:3306/int_tests");
```

Comment:
do we want to propagate our url in github?

Reply:
Fixed to get host from System.properties